web/app.js

```diff
@@ -2077,6 +2077,16 @@ const PDFViewerApplication = {
     return newFactor;
   },
 
+  _centerAtPos(previousScale, x, y) {
+    const { pdfViewer } = this;
+    const scaleDiff = pdfViewer.currentScale / previousScale - 1;
+    if (scaleDiff !== 0) {
```

Comment:
nit: check that the values are different before the computation. It probably doesn't matter that much but if we can avoid a useless division and subtraction.

Reply:
How about the updated patch instead, since it'll prevent the new method from being called in the first place if we're not actually zooming?

Reply:
This check might still make sense though when we've reached the MIN/MAX allowed scale values, which probably isn't common enough to worry about here?